### ISSUE TITLE
Normalize does not set <main> display block

### ISSUE THREAD

**Issue by nicolaiwolko** [zurb/foundation-sites#11577]
## What should happen?

normalize should set the `<main>` tag to `display: block`,
like it has done at version 6.4.4-rc1 with normalize.scss

## What happens instead?

Nothing is set for the `<main>` tag. IE is known for not implementing/support
the main tag, so at least `display: block` have to be set to make it behave
like a div.

## Possible Solution

- Consider using another vendor normalize
- Introduce your own normalization with:
`
main {
  display: block;
}
`

## Test Case and/or Steps to Reproduce (for bugs)

Use a `<main>` tag in IE with foundation and add for instance a background color. 
Place some content inside the main and the background color will not be shown.

**Comment by DanielRuf** [comment]
https://github.com/necolas/normalize.css/blob/master/normalize.css#L32
https://github.com/csstools/normalize.css/blob/master/normalize.css#L50

Can you provide a codepen which reproduces this?

**Comment by DanielRuf** [comment]
https://github.com/necolas/normalize.css/blob/master/normalize.css#L32
https://github.com/csstools/normalize.css/blob/master/normalize.css#L50

It seems this was fixed in normalize 8.0.1, see https://github.com/necolas/normalize.css/blame/8.0.1/normalize.css#L32 and https://github.com/necolas/normalize.css/commit/df07c00a92c683e591b68bcc45075ff7253b7482

I propose that we switch to csstools/normalize.css which is the right normalize imo.

**Comment by nicolaiwolko** [comment]
You are right, you are using normalize 8.0.0 at the moment:
https://github.com/zurb/foundation-sites/blob/develop/scss/vendor/normalize.scss#L2

So updating should fix it too

**Comment by DanielRuf** [comment]
Well, in the long run, updating is not the solution. We have to migrate to csstools/normalize.css. See https://github.com/csstools/normalize.css/issues/3#issuecomment-421401639 and https://github.com/necolas/normalize.css/issues/664

**Comment by jonathantneal** [comment]
> more followed and maintained

Would you quantify that? I maintained the necolas version while he walked away from the project. I’ll admit that it’s hard to follow updates to that version since I’m blocked from the project.

**Comment by ncoden** [comment]
@jonathantneal I corrected my comment.

**Comment by necolas** [comment]
> normalize.css is a project by Jonathan Neal, co-created with Nicolas Gallagher.

Please stop your BS. The reason you're blocked is because you gaslight me.

**Comment by ncoden** [comment]
@jonathantneal @jonathantneal

Hello Guys, I'm happy to see you two there, but at first I have to make this very clear:

You will not start another fight.

**Comment by jonathantneal** [comment]
I’m really sorry to read that, Nicolas. I would love, love to be on good terms again. I understand if you’re not interested at this time, and I do hope it changes. Would you consider unblocking me from the project if I agree not to comment on any issues until you say it’s okay? It really sucks to not be un-included like that.

As for this thread; y’all pick the libraries that serves your needs best.

I think the issue of `body { margin: 0 }` is a kind of tabs vs spaces thing. I tend to think normalize should stick to fixes, while many other intelligent people think certain opinions are practical.

I would have personally expected people to “die on a hill” for adding `* { box-sizing: border-box }` before preserving `body { margin: 0 }` but the later has obviously had a major influence on the project. I didn’t expect that.

@ncoden, totally understood. Please delete this message if you feel it’s aggravating.

**Comment by jonathantneal** [comment]
@necolas, I just saw your updated comment. Any chance we can talk? My email is jonathantneal@hotmail.com. I’d love a voice call, just incase our typed words come across the wrong way to each other.

**Comment by ncoden** [comment]
If I may continue with the purpose of this issue and the problem that Foundation is facing.

We care more about our developer's best interests than the struct following of the "browser normalization" approach. But on the other side, we want to rely on a watched and well-maintained project.

@necolas What is the maintenance status of `necolas/normalize.css`? Is there someone that currently ensure a technological watch on the project? As you can see, [many improvments](https://github.com/csstools/normalize.css/blob/master/CHANGELOG.md) were provided in the csstools' version this year. Would you consider adding them to your version?

@jonathantneal Developers generally don't like separations in communities and projects, and most often it doens't end very well. Do you think that creating a second `normalize.css` project is the best solution? Would there be others options that would be in the best interest of the whole community (for example at Foundation we generate several `.css` so the user can pick the one that fit the best its needs) ?

**Comment by jonathantneal** [comment]
@ncoden, I will think more about that. At the moment, I do not like having 2 separate projects. I would prefer the “opinionated” and “fixes only” versions both live in the same repo. I can add the opinionated version to csstools if you think that would be helpful.

I want to be sensitive to others who think the projects should be separate with separate names. For instance, there is @sindresorhus’ excellent and well-known version https://github.com/sindresorhus/modern-normalize/. How much of a name change works?

In the meantime, could you take any fixes from mine and make them PRs on necolas’ version if they are not already? That should make it easier to use his version and end any immediate concerns.

**Comment by ncoden** [comment]
> I want to be sensitive to others who think the projects should be separate with separate names.

> In the meantime, could you take any fixes from mine and make them PRs on necolas’ version if they are not already? That should make it easier to use his version and end any immediate concerns.

IMHO these 2 solutions in the same time would be the best. Using an other name to avoid confusions, and importing each project advantages into the other. I hope that this conflict will be resolved one day, but until then we would have two distinguishable projects, one with opinionated rules and the other without.

**Comment by jonathantneal** [comment]
Would you mind helping me with the name? Naming things is hard. Right now it’s @csstools/normalize.css on npm and the README.md just calls it normalize.css, since they have the same initial git history.

I’ve immediately updated the README.md to call it csstools-normalize (following the sindresorhus convention) and I’ve added the opinionated body style. I can add the others after referencing the current necolas release.

Anything else I can do to help?

**Comment by ncoden** [comment]
@DanielRuf What do you think?

**Comment by ncoden** [comment]
@jonathantneal The best you could do is a `csstools/normalize` version has a distinguishable name and a stable version that include an "opinionated" option (with all the rules from the @necolas' version, not something made in a rush).

Once this is done, released and stable, we could switch to `csstools/normalize`. But I'd like to get the opinon of @DanielRuf on this. @necolas I'd love to have your opinon too.

**Comment by jonathantneal** [comment]
He’s asked not to be mentioned in this thread. If you could edit your comment, I will delete this one.

**Comment by necolas** [comment]
> Please stop mentioning me in this thread

**Comment by ncoden** [comment]
@jonathantneal There is something I did not understood: What is the difference between [csstools-normalize](https://github.com/csstools/normalize.css) and [sanitize.css](https://github.com/csstools/sanitize.css) ?

**Comment by jonathantneal** [comment]
csstools’ normalize is just browser fixes, while sanitize is normalize plus opinionated changes based on developer preferences — things that will never ship as the default — things like `box-sizing: border-box`.

**Comment by ncoden** [comment]
@jonathantneal What is the difference with necolas' normalize then?

**Comment by jonathantneal** [comment]
That normalize includes 2 opinionated styles — font styles on inputs and margin removal on the body.
